src/timepicker/timepicker.component.ts

```diff
@@ -65,11 +73,28 @@ export class TimePicker {
 
 	@Output() valueChange: EventEmitter<string> = new EventEmitter();
 
+	writeValue(obj: any): void {
+		this.value = obj;
+	}
+	registerOnChange(fn: any): void {
```

Comment:
generally we use full names ... 
```suggestion
	registerOnChange(callback: any) {
```

(ditto for registerOnTouched)